LEAF_Nexus/auth_domain/index.php

```diff
@@ -103,11 +103,15 @@
         }
         else
         {
-            echo 'Unable to log in: User not found in global database.';
+            header('Refresh: 4;URL=' . $_SERVER["HTTP_REFERER"]);
```

Comment:
HTTP_REFERER is not very reliable, this should instead use a relative path to ../login/index.php. Check syntax for this.

Reply:
Addressed in commit `134`